### PR TITLE
Testing with rspec

### PR DESCRIPTION
**Pull request by lucaasdfg**

### CODE REVIEW

File: spec/enumerables_spec.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../enumerables.rb'
+describe Enumerable do
+  array = [1, 2, 3, 4, 5]
+  falsy_array = [1, 22, false, 0, 'oop']
```
Comment (maelfosso) [inline spec/enumerables_spec.rb]: Not good !!!
Do
```ruby
let(array) {[1,2,3,4,5]}
...
```

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required

Hello @lucaasdfg 

Awesome, Amazing. Nothing to add except only one thing **Constant declaration**
For constant declarations, please use `let`. Read the comment to know how.

Kindly make the change and submit a new code review request.
Thanks

**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com) | [LinkedIn](https://www.linkedin.com/in/mael-elvis-fosso-650b6346/)

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**You have done a great job on this on this project, however, we still have one thing to update! ⬇️** 

- Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by ebukaume — APPROVED** [review]
## Approved! :champagne: 

Hi @lucaasdfg and @alexawesomecode :wave: 

The README now looks more descriptive :+1: 

**You can go ahead and merge this branch**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**